xtensor-blasConfig.cmake.in

```diff
@@ -22,10 +22,9 @@ if(NOT TARGET @PROJECT_NAME@)
   get_target_property(@PROJECT_NAME@_INCLUDE_DIRS xtensor-blas INTERFACE_INCLUDE_DIRECTORIES)
   find_dependency(BLAS REQUIRED)
   find_dependency(LAPACK REQUIRED)
-  target_link_libraries(xtensor-blas INTERFACE ${BLAS_LIBRARIES} ${LAPACK_LIBRARIES})
-  if(${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.8)
-    target_compile_features(xtensor-blas INTERFACE cxx_std_14)
-  endif()
```

Comment:
This should stay!

Reply:
This is exported in the targets file.

Reply:
(A clarification: here you are constructing a new target if it does not exists)

Reply:
Ah, sorry, I did read the file carefully enough. Indeed, it comes from the targets file.